ansible-ipi-install/roles/scale-worker/tasks/30_scale_machinesets.yml

```diff
@@ -0,0 +1,39 @@
+- name: Register machineset name
+  environment:
+    KUBECONFIG: "{{ kubeconfig_file }}"
+  shell: |
+    oc -n openshift-machine-api get machineset  | awk '/worker/ {print $1}'
+  register: worker_machineset
+  failed_when: worker_machineset.stdout == ''
+
+- name: Scale machineset - {{ worker_machineset.stdout }}
```

Comment:
I think we can scale machineset before " Wait for BareMetalHost nodes to get into Ready state" task in 20_create_bmh.yml so that if we are scaling up by two workers and one nodes fails introspection, at least the other node will be added to the cluster..thoughts? 
@mukrishn